test/spark/docker-compose.yaml

```diff
@@ -0,0 +1,88 @@
+version: '3.9'
+services:
+  postgres:
+    image: "postgres:11"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: lakefs
+      POSTGRES_PASSWORD: lakefs
+  lakefs:
+    image: "${REPO}/lakefs:${TAG}"
+    ports:
+      - "8000:8000"
+    networks:
+      default:
+        ipv4_address: 10.5.0.55
+    depends_on:
+      - "postgres"
+    volumes:
+      - ./lakectl-tester.yaml:/home/lakefs/.lakectl.yaml
+      - .:/local
+    environment:
+      - LAKEFS_GATEWAYS_S3_DOMAIN_NAME=s3.docker.lakefs.io
+      - LAKEFS_AUTH_ENCRYPT_SECRET_KEY=some random secret string
+      - LAKEFS_DATABASE_CONNECTION_STRING=postgres://lakefs:lakefs@postgres/postgres?sslmode=disable
+      - LAKEFS_BLOCKSTORE_LOCAL_PATH=/home/lakefs
+      - LAKEFS_BLOCKSTORE_TYPE
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY
+      - LAKEFS_LOGGING_LEVEL=DEBUG
+      - LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
+      - LAKEFS_STATS_ENABLED
+      - LAKEFS_CATALOGER_TYPE
+      - LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCOUNT
```

Comment:
Please cleanup gcp/azure args.

Reply:
will pass env from github workflow later when we like to run the same with different sets of block adapters